lib/i18n/supportedLanguages.ts

```diff
@@ -1,7 +1,15 @@
 /**
  * Supported languages of the library
  */
-export const supportedLanguages = ['en', 'fr', 'es', 'cn', 'ja', 'de'] as const
+export const supportedLanguages = [
+  'en',
+  'fr',
+  'es',
+  'cn',
+  'ja',
+  'de',
+  'ptBr',
```

Comment:
I'm wondering if this should be `pt-br` instead.

Reply:
@lbenie Thinking now, is a better approach and a more common one.